ci-operator/step-registry/cucushift/installer/check/nutanix/zones/cucushift-installer-check-nutanix-zones-commands.sh

```diff
@@ -13,17 +13,35 @@ declare prism_element3
 # shellcheck source=/dev/null
 source "${NUTANIX_AUTH_PATH}"
 
-node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
-
 if [[ "$SINGLE_ZONE" == "true" ]]; then
+    node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
     echo "Check all nodes running on single zone: $prism_element1"
     for zone in ${node_zone_list}; do
         if [ "$zone" != "$prism_element1" ]; then
             echo "Fail: fail to check single zone: $zone, expected: $prism_element1"
             exit 1
         fi
     done
+elif [[ "$COMPUTE_ZONE" != "" ]]; then
+    compute_pes=$(echo "$COMPUTE_ZONE" | sed -e "s/failure-domain-1/Development-STS/g" -e "s/failure-domain-2/Development-zonal/g" -e "s/failure-domain-3/Development-zonal2/g")
+    compute_zone_list=$(oc get nodes -l node-role.kubernetes.io/worker= -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
+    for zone in ${compute_zone_list}; do
+        if ! [[ $compute_pes =~ $zone ]]; then
```

Comment:
Here once set compute zones, can do an exact match (e.g: `==`) to run the check?

Reply:
`compute_pes` is a list, here it is `Development-STS Development-zonal`, do you want to iterate it to match exactly?

Reply:
What does `compute_zone_list` look like?

Reply:
`Development-STS Development-zonal` or `Development-STS Development-zonal`, that's depend on cluster installed I think.

Reply:
I think they can do an exact match by something like:
```
$ echo "${compute_zone_list}" | tr " " "\n" | sort -u | xargs
Development-STS Development-zonal

$ echo "${compute_pes}" | tr " " "\n" | sort -u | xargs
Development-STS Development-zonal
```